### PR TITLE
Add CONTRIBUTING.md and update CI integration links

### PR DESCRIPTION
**Pull request by creviera**
# Description

Update description of project, add a CONTRIBUTING.md file and info on how to get help, and update CI project links.

# Test

Make sure links work and review language for spelling, grammar, accuracy, curse words, etc.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## How to Contribute to SecureDrop Client
+
+The SecureDrop Client is a desktop application for journalists to communicate with sources and work with SecureDrop submissions on the [SecureDrop Workstation](https://github.com/freedomofpress/securedrop-workstation).
+
+Check out our [Development Roadmap](https://github.com/freedomofpress/securedrop/wiki/Development-Roadmap) to see plans and priorities listed under "SecureDrop Workstation." Also, see our list of [good first issues](https://github.com/freedomofpress/securedrop-client/labels/good%20first%20issue) and issue labeled [help wanted](https://github.com/freedomofpress/securedrop-client/labels/help%20wanted).
```
Comment (eloquence) [inline CONTRIBUTING.md:5]: issue labeled->issues labeled

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+If have any questions or need help, you can reach out to us in our public [Gitter channel](https://gitter.im/freedomofpress/securedrop) or [public Discourse forum](https://forum.securedrop.org/).
```
Comment (eloquence) [inline CONTRIBUTING.md:7]: If have->If you have

File: README.md [hunk 3]
```diff
@@ -1,11 +1,18 @@
...
+To learn more about architecture and our rationale behind using a Qubes OS approach for source communication and viewing submissions, see the [SecureDrop Workstation readme](https://github.com/freedomofpress/securedrop-workstation/blob/main/README.md).
 
-**IMPORTANT:** This project is in alpha and should not be used in production environments. There are known bugs which can be found in this project’s issue tracker.
+**IMPORTANT:** This project is currently undergoing a pilot sutdy and should not be used in production environments.
```
Comment (eloquence) [inline README.md:15]: sutdy->study

**Comment by creviera** [comment]
This is a WIP until we someone with GitHub org admin privs adds this integration https://codecov.io/gh/freedomofpress/securedrop-client (we should also discuss granting `sd-maintainers` admin privs here tomorrow).

**Comment by eloquence** [comment]
Content LGTM, flagged a couple of minor typos. Thanks for making the repo more contributor-friendly!

**Comment by creviera** [comment]
This is ready for review now that we've decided against adding codecov.

**Review by emkll — APPROVED** [review]
thank you @creviera !

**Pull request merged**